antora-ui-camel/src/css/doc.css

```
@@ -634,3 +634,9 @@ kbd,
 .stretch {
   width: 100%;
 }
+/* for mobile screens */
+@media screen and (max-width: 600px) {
+  .doc aside {
+    width: auto;
+  }
+}
```

Perhaps we could remove the categories on any screen smaller than 1024px, like this:

```suggestion
@media screen and (max-width:1023px) {
  .doc aside {
    display: none;
  }
}
```

That way we don't introduce any new media breakpoints.